<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="topV">
      <item>
       <widget class="QLabel" name="foodCountLbl">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Food delivered: 0 out of 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="topH">
        <item>
         <layout class="QVBoxLayout" name="sim">
          <item>
           <widget class="QGroupBox" name="simBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Simulation</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="simCtrlBox">
               <property name="title">
                <string>Control</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="initSimBtn">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Initialize</string>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="startSimBtn">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Start</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stopSimBtn">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Stop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Simulation speed</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDial" name="speedDial">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="simParamBox">
               <property name="title">
                <string>Parameters</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="antsLbl">
                  <property name="text">
                   <string>Ants</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="antsSB"/>
                </item>
                <item>
                 <widget class="QLabel" name="phStrengthLbl">
                  <property name="text">
                   <string>Pheromone strength</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="phStrengthSl">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksBelow</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="phSpreadLbl">
                  <property name="text">
                   <string>Pheromone spread</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="phSpreadSl">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>5</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksBelow</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="phDecayLbl">
                  <property name="text">
                   <string>Pheromone decay</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="phDecaySl">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksBelow</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="scene">
          <item>
           <widget class="QGraphicsView" name="graphicsView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="dragMode">
             <enum>QGraphicsView::ScrollHandDrag</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="zoomSlider">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>350</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="gen">
          <item>
           <widget class="QGroupBox" name="caveBox">
            <property name="title">
             <string>Cave Generation</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="seedLbl">
               <property name="text">
                <string>Seed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="seedSB"/>
             </item>
             <item>
              <widget class="QLabel" name="thresholdLbl">
               <property name="text">
                <string>Threshold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="thresholdSB"/>
             </item>
             <item>
              <widget class="QLabel" name="rockRatioLbl">
               <property name="text">
                <string>Rock ratio</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="rockRatioSB"/>
             </item>
             <item>
              <widget class="QLabel" name="stepsLbl">
               <property name="text">
                <string>Steps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="stepsSB"/>
             </item>
             <item>
              <widget class="QLabel" name="radiusLbl">
               <property name="text">
                <string>Radius</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="radiusSB"/>
             </item>
             <item>
              <widget class="QRadioButton" name="mooreRad">
               <property name="text">
                <string>Moore</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="neumannRad">
               <property name="text">
                <string>Neumann</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="generateCaveBtn">
               <property name="text">
                <string>New Cave</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
